research-software-directory/RSD-as-a-service
# Update GitLab logo

# Update GitLab logo

Fixes #318

Changes proposed in this pull request:

* Update the GitLab logo

How to test:

* create a new software entry with a GitLab URL or set GitLab manually
* in the software page the new logo should be displayed (maybe a complete reload is also needed in the browser)

PR Checklist:
